src/palimpzest/operators/physical.py

```diff
@@ -4,9 +4,10 @@
 from palimpzest.corelib.schemas import ImageFile
 from palimpzest.datamanager import DataDirectory
 from palimpzest.elements import *
-from palimpzest.elements import Any
-from palimpzest.solver import Solver
-from palimpzest.tools.profiler import Profiler
+from palimpzest.elements import Any # TODO: can we delete?
```

Comment:
In terms of overall code cleanliness, this is my least favorite file in the project.  Let's consider how we might redo the class hierarchy and reduce the amount of code needed for each new physical operator.

Reply:
100% agreed, the simplest thing I can think of is to have the parallel induce and filter operations subclass their non-parallel counterparts, but even that only brings things from ~1000 lines --> ~900. There's definitely more we could to to clean this up.